src/net/java/sip/communicator/impl/protocol/jabber/extensions/jitsimeet/JsonMessageExtension.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Jitsi, the OpenSource Java VoIP and Instant Messaging client.
+ *
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.java.sip.communicator.impl.protocol.jabber.extensions.jitsimeet;
+
+import org.jivesoftware.smack.packet.*;
+import org.jivesoftware.smack.provider.*;
+import org.xmlpull.v1.*;
+
+/**
+ * An implementation of a {@link PacketExtension} for json messages.
+ *
+ * @author Praveen Kumar Gupta
+ */
+public class JsonMessageExtension
+        implements ExtensionElement
```

Comment:
Hey Praveen,

Good work!

Can you please reuse AbstractPacketExtension (and also the default provider) here? You can use this as an example:
https://github.com/jitsi/jitsi/pull/483/commits/bfebd504dfaf5e9e1e25e0a57a731f5f2de3ca2f

Reply:
Yeah, will update it. :+1:

Reply:
One more minor thing. Can you please use this for the namespace:
`http://jitsi.org/jitmeet`